reference/twig_reference.rst

```diff
@@ -134,11 +134,30 @@ asset_version
 
 .. code-block:: twig
 
-    {{ asset_version(packageName = null) }}
+    {{ asset_version(path, packageName = null) }}
 
+``path``
+    **type**: ``string``
 ``packageName`` *(optional)*
     **type**: ``string`` | ``null`` **default**: ``null``
 
+.. code-block:: yaml
```

Comment:
I would not add these two code blocks. We should keep the reference as short as possible.